blockmanager.go

```diff
@@ -2547,3 +2549,61 @@ func (b *blockManager) findPrevTestNetDifficulty(hList headerlist.Chain) (uint32
 	}
 	return lastBits, nil
 }
+
+// onBlockConnected queues a block notification that extends the current chain.
+func (b *blockManager) onBlockConnected(header wire.BlockHeader, height uint32) {
+	select {
+	case b.blockNtfnChan <- blockntfns.NewBlockConnected(header, height):
+	case <-b.quit:
+	}
+}
+
+// onBlockDisconnected queues a block notification that reorgs the current
+// chain.
+func (b *blockManager) onBlockDisconnected(headerDisconnected wire.BlockHeader,
+	heightDisconnected uint32, newChainTip wire.BlockHeader) {
+
+	select {
+	case b.blockNtfnChan <- blockntfns.NewBlockDisconnected(
+		headerDisconnected, heightDisconnected, newChainTip,
+	):
+	case <-b.quit:
+	}
+}
+
+// Notifications exposes a receive-only channel in which the latest block
+// notifications for the tip of the chain can be received.
+func (b *blockManager) Notifications() <-chan blockntfns.BlockNtfn {
+	return b.blockNtfnChan
+}
+
+// NotificationsSinceHeight returns a backlog of block notifications starting
+// from the given height to the tip of the chain.
+func (b *blockManager) NotificationsSinceHeight(
```

Comment:
Currently, there's no way for the caller to signal that: they don't know the current height, and don't care about any backlog notifications.

Reply:
Hah thought I had a check for this already. Fixed.